### PR TITLE
[chrome-ext] Add chrome extension to circle config

### PR DESCRIPTION
**Pull request by styu**
@vsiao PR-ing this into your branch

adding it to circle config. we can add lint in a FLUP :)

### CODE REVIEW

**Comment by styu** [comment]
Oops I’ll remake this

File: chrome-extension/config.circle.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+        apiKey: "asdf",
+        authDomain: "asdf",
+        databaseURL: "asdf",
+        storageBucket: "asdf"
+    },
+    slack: {
+        clientId: "asdf",
+        clientSecret: "asdf"
+    }
+};
```
Comment (vsiao) [inline chrome-extension/config.circle.js]: can' t just use the example one to keep them in sync?

Comment (styu) [reply chrome-extension/config.circle.js]: sure i can do that

Comment (styu) [reply chrome-extension/config.circle.js]: actually, there's no example one?

Comment (vsiao) [reply chrome-extension/config.circle.js]: oh, doh. we should make one

Comment (styu) [reply chrome-extension/config.circle.js]: mk, we can do that in a follow up

**Pull request merged**